scout/server/blueprints/variant/views.py

```diff
@@ -292,58 +292,23 @@ def acmg():
 @templated("variant/clinvar.html")
 def clinvar(institute_id, case_name, variant_id):
     """Build a clinVar submission form for a variant."""
-    data = clinvar_export(store, institute_id, case_name, variant_id)
+
     if request.method == "GET":
+        data = clinvar_export(store, institute_id, case_name, variant_id)
```

Comment:
It now becomes rather apparent that this should be two different endpoints, one for GET, one for POST. But, with all request processing as well in the controller it is already so easy to read it hardly matters. Laters if at all.